teambit.bvm/commands/install/install.ts

```diff
@@ -14,6 +14,13 @@ export class InstallCmd implements CommandModule {
       default: 'latest',
       type: 'string'
     })
+    .option({
+      updatePath: {
```

Comment:
as the default is true, I would consider changing it to the negative way. like skip-update-path.
but I'm open to hear arguments about this way as well

Reply:
It will be `--no-update-path` in this case. The only problem is that in help it shows `---updatePath [Default: true]`